Language/Structure/Compound Operators/increment.adoc

```diff
@@ -63,12 +63,19 @@ y = x++;      // x contains 4, but y still contains 3
 === Notes and Warnings
 [%hardbreaks]
 
+--
+// HOW TO USE SECTION ENDS
+
+
+
+
+// SEE ALSO SECTION ENDS
+[#see_also]

```

Comment:
@kengdahl you are not adding the two dashes (--) under this, please review the PRs